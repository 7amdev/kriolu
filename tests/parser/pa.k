pa (mimoria i = 0; i < 2; i = i + 1) {
    imprimi "Hi";
}