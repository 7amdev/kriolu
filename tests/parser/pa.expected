<pa>
  <initializer>
    <mimoria>
      i
      0.0
  <condition>
    ((get i) < 2.0)
  <increment>
    (i = ((get i) + 1.0))
  <body>
    <block>
      <imprimi>
        'Hi'